sys/linux/bpf.txt

```diff
@@ -400,16 +408,21 @@ bpf_obj_get {
 	file_flags	flags[bpf_open_flags, int32]
 }
 
+bpf_attach_targets [
+	cgroup	fd_cgroup[opt]
+	map	fd_bpf_map[opt]
```

Comment:
AFAIU, this means syzkaller will try to attach to BPF maps about half the time. Since it should only work for sockmaps, maybe we should specialize `fd_bpf_map` here to avoid spending time trying to attach to e.g. random hashmaps? That would require a sockmap-specialized version of `bpf$MAP_CREATE` though (with a subset of values for `map_type`).